Changelog.md

```diff
@@ -1,5 +1,38 @@
 gfxutil changelog
 =================
+#### 1.79b joevt Dec 24, 2019
```

Comment:
Could you make it 1.80b without suffixes as we now mainline the changes.

Reply:
Fixed.